cluster/examples/kubernetes/ceph/operator-with-csi.yaml

```diff
@@ -0,0 +1,574 @@
+apiVersion: v1
```

Comment:
Could we just include the csi properties in operator.yaml instead of having a separate yaml?

Reply:
sure